test/rfqm_test.ts

```diff
@@ -97,12 +101,17 @@ describe.skip(SUITE_NAME, () => {
             rfqBlockchainUtilsMock.generateMetaTransaction(anything(), anything(), anything(), anything(), anything()),
         ).thenReturn(MOCK_META_TX);
         const rfqBlockchainUtils = instance(rfqBlockchainUtilsMock);
+
+        const connection = await getDBConnectionAsync();
```

Comment:
we might need to reset the DB at the end to avoid state affecting other tests

Reply:
I also think it's worth testing an entity read and write to make sure values are being stored and read as expected

Reply:
Added a read from the DB at the end!

Will add a query to Truncate the table that was added to.